[2024-12-01T13:00:07.724+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-01T13:00:07.765+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: send_csv_to_kafka.send_csv_to_kafka manual__2024-12-01T12:59:59.934637+00:00 [queued]>
[2024-12-01T13:00:07.792+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: send_csv_to_kafka.send_csv_to_kafka manual__2024-12-01T12:59:59.934637+00:00 [queued]>
[2024-12-01T13:00:07.805+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-01T13:00:07.886+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): send_csv_to_kafka> on 2024-12-01 12:59:59.934637+00:00
[2024-12-01T13:00:07.917+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=905) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-01T13:00:07.928+0000] {standard_task_runner.py:72} INFO - Started process 919 to run task
[2024-12-01T13:00:07.923+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'send_csv_to_kafka', 'send_csv_to_kafka', 'manual__2024-12-01T12:59:59.934637+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmp6nzs3uf3']
[2024-12-01T13:00:07.941+0000] {standard_task_runner.py:105} INFO - Job 43: Subtask send_csv_to_kafka
[2024-12-01T13:00:08.324+0000] {task_command.py:467} INFO - Running <TaskInstance: send_csv_to_kafka.send_csv_to_kafka manual__2024-12-01T12:59:59.934637+00:00 [running]> on host 45770b1a429e
[2024-12-01T13:00:09.033+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='send_csv_to_kafka' AIRFLOW_CTX_TASK_ID='send_csv_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2024-12-01T12:59:59.934637+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-01T12:59:59.934637+00:00'
[2024-12-01T13:00:09.057+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-01T13:00:09.172+0000] {dag_test.py:86} ERROR - Error in Kafka task: Default value <None> must match schema type: string
[2024-12-01T13:00:09.174+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-01T13:00:09.214+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-01T13:00:09.216+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=send_csv_to_kafka, task_id=send_csv_to_kafka, run_id=manual__2024-12-01T12:59:59.934637+00:00, execution_date=20241201T125959, start_date=20241201T130007, end_date=20241201T130009
[2024-12-01T13:00:09.331+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-01T13:00:09.395+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-01T13:00:09.400+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
